README.md

```diff
@@ -22,28 +87,41 @@ Contents
 npm install --save-dev @manta-style/cli
 ```
 
-You could also install it globally, which adds a command line tool `ms` to your system.
+To install globally
 
-### Plugins
+```
+npm install -g @manta-style/cli
+```
 
-Manta Style needs plugins to support different file types and generate mock data.
+This adds a command line tool `ms` to your system.
 
-The example of Quick Start below is using TypeScript. So first you might want to install TypeScript support to Manta Style.
+### Builders
+
+Manta Style supports generation of mock data from different typing systems via binding builders.
+
+In [Quick Start](#quick-start) we show an example using TypeScript. You will need to [install TypeScript](https://www.typescriptlang.org/index.html#download-links) _and_ add the Manta Style TypeScript builder:
 
 ```sh
 npm install --save-dev @manta-style/plugin-builder-typescript
 ```
 
-If you are new to Manta Style, please install the plugins below. We are going to use them in [Quick Start](#quick-start).
+### Plugins
 
-```sh
-npm install --save-dev @manta-style/plugin-mock-example @manta-style/plugin-mock-faker
-```
+Manta Style supports a handful of data generating features via plugins.
+One major example is `@manta-style/plugin-mock-faker`. This is a plugin that enables you to generate fake datas that look real.
 
 You could check [Plugins](#plugins) for the usages of official plugins. You could make our own plugins as well.
 
+<!-- TODO: add support for creating plugins? -->
```

Comment:
yeah, i think @tanhauhau and me can provide more information for creating plugins.